site/content/faq/how-can-you-use-zap-to-scan-apis.md

```diff
@@ -6,14 +6,15 @@ weight: 7
 ---
 
 
-ZAP understands API formats like JSON and XML and so can be used to scan APIs.
+ZAP understands API formats like JSON and so can be used to scan APIs.
+Earlier ZAP understanded XML format like WSDL. The SOAP Scanner add-on has been removed for the time being.
 
 The problem is usually how to effectively explore the APIs.
 
 There are various options:
 
   * If your API has an OpenAPI/Swagger definition then you can import it using the [OpenAPI add-on](/docs/desktop/addons/openapi-support/).
-  * If your API has a WSDL then you can import it using the SOAP Scanner add-on.
```

Comment:
Once can still use the SOAP add-on, it's "just" not available in the marketplace. Maybe we should add a link here with the remarks about the issues it has.

Reply:
> One can still use the SOAP add-on, it's "just" not available in the marketplace. Maybe we should add a link here with the remarks about the issues it has.
It has some problem to setup add-on that not available in marketplace for DockerZAP and autotests. This requires volume directory with add-os and stores this in remote git and pulling for every host where working ZAP.

Reply:
Agreed, I'm not saying that's straightforward to get it (compared to marketplace) just that you can still use it (which the removed statement states).

Reply:
If we dont document how to use it then can we really expect anyone to use it? I think we should either document how people can use it or accept that its essentially no longer available. Or convince someone to fix it of course :D

Reply:
The blog post linked later shows how to use it, as long as you get the add-on (core help documents from where they are loaded).